sos/plugins/openstack_placement.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) 2019 Red Hat, Inc., Lee Yarwood <lyarwood@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenStackPlacement(Plugin):
+    """OpenStack Placement
+    """
+    plugin_name = "openstack_placement"
```

Comment:
How is this going to work for older distributions that still ship the pre-extraction package layout? Remember that sos needs to support all possible shipping versions until the old ones are completely removed. For the time being, unless you are absolutely certain the old ones are gone from the field, there may be a need to support both.

Reply:
This still works with earlier releases because I've not removed any code from nova plugin. I've tested it against Queens (OSP13) and it seams working well, it collects all placement logs and info. If you can think of any test and release I should test against, please let me know.